modules/sd_disable_initialization.py

```diff
@@ -65,7 +65,7 @@ def create_model_and_transforms_without_pretrained(*args, pretrained=None, **kwa
             return self.create_model_and_transforms(*args, pretrained=None, **kwargs)
 
         def CLIPTextModel_from_pretrained(pretrained_model_name_or_path, *model_args, **kwargs):
-            res = self.CLIPTextModel_from_pretrained(None, *model_args, config=pretrained_model_name_or_path, state_dict={}, **kwargs)
+            res = self.CLIPTextModel_from_pretrained(pretrained_model_name_or_path, *model_args, config=pretrained_model_name_or_path, state_dict={}, **kwargs)

```

Comment:
`config=pretrained_model_name_or_path` can be removed since it has the same behavior.

See: https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_utils.py#L3579
This is the only place where it is used as a string/path and in the next line it gets replaced anyway.